src/symtable/SymbolTableScope.ts

```diff
@@ -255,7 +254,7 @@ export class SymbolTableScope {
             if (bound === null) {
                 this.errorAtDirective(name, "nonlocal declaration not allowed at module level");
             }
-            if (bound.has(name)) {
+            if (!bound.has(name)) {
```

Comment:
ooops

Reply:
haha thank heavens for tests! this could've taken us hours in a few months to fix